modules/appeals_api/lib/appeals_api/hlr_pdf_constructor.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class HlrPdfConstructor
+    PDF_TEMPLATE = Rails.root.join('modules', 'appeals_api', 'config', 'pdfs')
+
+    def initialize(higher_level_review_id)
+      @higher_level_review = AppealsApi::HigherLevelReview.find(higher_level_review_id)
+      veteran
+    end
```

Comment:
why pass in the id and not the HigherLevelReview object?

Reply:
I am generally opposed to passing persisted objects between different classes. While unlikely in this case, when using jobs, it is possible for the underlying database record to change between steps and this makes sure this class starts with the exact DB record.